RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -6189,7 +6252,19 @@ boolean auto_check_conditions(string conds)
 				if(req_loc == $location[none])
 					abort('"' + condition_data + '" does not properly convert to a location!');
 				return my_location() == req_loc;
-			// data: <propname><comparison operator><value>
+			// data: <propname><comparison operator><integer value>
+			// As a precaution, autoscend aborts if to_location returns $location[none]
+			case "turnsspent":
+				matcher m6 = create_matcher("([^=<>]+)([=<>]+)(.+)", condition_data);
+				if(!m6.find())
+					abort('"' + condition_data + '" is not a proper turnsspent condition format!');
+				location loc = to_location(m6.group(1));
+				if(loc == $location[none])
+					abort('"' + condition_data + '" does not properly convert to a location!');
+				if(!($strings[=,==] contains m6.group(2)))
+					return compare_numbers(loc.turns_spent, m6.group(3).to_int(), m6.group(2));
+				return loc.turns_spent == m6.group(3).to_int();
+			// data: <location><comparison operator><value>
```

Comment:
```suggestion
			// data: <propname><comparison operator><value>
```

Reply:
this is the comment for case "prop" so it should be fine